### ISSUE TITLE
i cant install tfr on arch linux

### ISSUE THREAD

**Issue by zx155** [Screetsec/TheFatRat#833]
this is a pic of bug or something
![Uploading 462542251_8970996399579341_1322477068562521611_n.png…]()

**Comment by zx155** [comment]
nah, sr guys, but here is are bug. First, they ask me to setup or install 2 times. i chose to install 2 times with the following error
xterm -> Not OK
dns-utils -> Not OK
Mono-Denvelop Utils -> Not OK
GCC -> OK
apache2 -> Not OK
gnome-terminal -> Not OK
upx-ucl -> Not OK
ruby -> Not OK
Python2 -> Not OK
Python3 -> OK
Python3-pip -> OK
Openssl -> OK
jarsigner from default-jdk-> Not OK
Unzip -> OK
keytool -> Not OK
Zipalign -> OK
mingw-w64 -> Not OK
mingw-32 -> Not OK
DX -> OK
Aapt -> OK
Apktool -> OK
Baksmali -> OK
Metasploit -> OK
Backdoor-factory -> Not OK
Searchsploit -> Not OK
Was not possible to install The Packages Labeled (Not Ok) in this list above
Try : (apt-get remove --purge <packagename> && apt-get autoremove && apt-get install -f)
before running fatrat setup script again

**Comment by peterpt** [comment]
you should check if you have any repositories in /etc/apt/sources.list configured , some basic tools are installed from your OS official repositories , only some others are from kali repository .

**Comment by NFS11** [comment]
what im more concerned about is how tf you successfully installed arch linux 💀

**Comment by crashtec** [comment]
Try parrot os

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
